practices/backlog_cleanup.md

```diff
@@ -0,0 +1,60 @@
+---
+slug: backlog-cleaning
+stages:
+  - development
+short_description: Cleaning the product backlog is essential to keep track of which key product features should be developed next. 
+tags:
+  - backlog cleaning
+  - agile
+  - scrum
+keywords:
+  - backlog
+  - cleaning
+  - grooming
+  - agile
+  - scrum
+author_github_username: cadamini
+---
+
+# Backlog Cleanup
+
+**TL;DR**
+
+A well-managed backlog results in a better overview of your important open issues and features, faster planning, and a cleaner scope. Periodically remove obsolete items from the backlog that have been created throughout the project lifecycle.
+
+## What Is Backlog Cleanup  
+
+Backlog Cleanup describes the process of managing issues or ideas for new features in your product backlog. After some strategy meetings in the last months/quarters, some of the items in your backlog may be out of scope, since the product strategy has shifted in a different direction.
+
+![Product backlog](/files/backlog_cleanup.webp)
+[Name](https://img.freepik.com/free-photo/close-up-sticky-note-with-product-backlog_23-2148513829.jpg)
```

Comment:
@cadamini Line 29 is the image itself, and line 30 should be a source of the image if licensed under some license that requires mentioning the author. (e.g. CC v4).

Please remove the line 30 or just set an URL to the article/page with the source image.

Reply:
```suggestion
![Product backlog](/files/backlog_cleanup.webp)
```

See above. Completely free image. Just remove this if this is ok.